docs-src/concepts/what-is-a-search-attribute.md

```diff
@@ -29,23 +29,28 @@ A Temporal Cluster has a set of default Search Attributes already available.
+| RunId                      | Keyword  | Identifies the current Workflow Execution Run.                                                                                                                               |
+| ExecutionDuration          | Int      | The time needed to run the Workflow Execution. Available only for closed Workflows.                                                                                          |
+| HistoryLength              | Int      | The number of events in the history of Workflow Execution. Available only for closed Workflows.                                                                              |
+| StateTransitionCount       | Int      | The number of times that Workflow Execution has persisted its state. Available only for closed Workflows.                                                                    |
+| TaskQueue                  | Keyword  | Task Queue used by Workflow Execution.                                                                                                                                       |
+| TemporalChangeVersion      | Keyword  | If Workflow versioning is enabled, list of change/version pairs will be stored here.                                                                                         |
+| BinaryChecksums            | Keyword  | List of binary Ids of Workers that run the Workflow Execution.                                                                                                               |
+| BatcherNamespace           | Keyword  | Used by internal batcher to indicate the Namespace where batch operation was applied to.                                                                                     |
+| BatcherUser                | Keyword  | Used by internal batcher to indicate the user who started the batch operation.                                                                                               |
+| NamespaceId                | Keyword  | Identifies the Namespace of the Search Attribute.                                                                                                                            |
+| TemporalScheduledStartTime | Datetime | The time that the Workflow is schedule to start at.                                                                                                                          |
+| TemporalScheduledById      | Keyword  | Identifies each scheduled Workflow by an Id.                                                                                                                                 |
+| TemporalSchedulePaused     | Boolean  | Indicates if the Schedule has been paused.                                                                                                                                   |
```

Comment:
Also, somehow make it clear that this attribute is set on _schedules_, not on workflows. (Schedules can have search attributes too)